plugin/vinegar.vim

```diff
@@ -36,6 +36,11 @@ nnoremap <silent> <Plug>VinegarTabUp :call <SID>opendir('tabedit')<CR>
 nnoremap <silent> <Plug>VinegarSplitUp :call <SID>opendir('split')<CR>
 nnoremap <silent> <Plug>VinegarVerticalSplitUp :call <SID>opendir('vsplit')<CR>
 
+augroup Vinegar_redraw
+  autocmd!
+  autocmd OptionSet wildignore windo exe "norm \<c-l>"
```

Comment:
Doesn't `windo` leave the last window focused?

Reply:
Yes, apparently the focus can change. Too bad. I do not know how to achieve this otherwise. I also do not like the visible redraw for my current configuration. Executing `<c-l>` in all windows looks like a glitch. For me solving this is actually more complicated than I would like to have it. I close this PR and see if I find the time for futher investigation. I also missed the guards for older vim versions without the event OptionSet.